content/projects/jenkins-operator/index.adoc

```diff
@@ -38,6 +39,7 @@ With declarative configuration and full lifecycle management based on the link:h
 === Contribution
 
 Any contributions to both codebase and documentation are more than welcome. See link:https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md[*Contribution Model*] for more details.
+Also see link:https://github.com/jenkinsci/kubernetes-operator#community[this page] for information about the project chat and regular meetings.
```

Comment:
I don't know enough about screen readers (but working on it), but I think this would read as "Also see this page for information about the project chat and regular meetings"

Maybe its worth being "Also see the community section of the github readme for more information about the project chat and regular meetings"?